infra/modules/azure_monorepo_single_env_starter_pack/id_infra.tf

```diff
@@ -0,0 +1,112 @@
+resource "azurerm_user_assigned_identity" "infra_ci" {
+  resource_group_name = azurerm_resource_group.main.name
+  location            = local.ids.location
+  name                = format(local.ids.infra_name, "ci")
+
+  tags = var.tags
```

Comment:
what about adding

````
lifecycle {
    prevent_destroy = true
  }
```

on every identity?

Reply:
We have an Azure Policy to automatically set a lock on managed identities